docs/path_handlers.rst

```diff
@@ -2,249 +2,252 @@
 .. slug: path-handlers
 .. author: The Nikola Team
 
-Nikola supports special links with the syntax ``link://kind/name``. In the templates you can also use ``_link(kind, name)``.
-You can also add query strings (``?key=value``) for extra arguments, or pass keyword arguments to ``_link`` in templates (support and behavior depends on path handlers themselves)
+Nikola supports special links with the syntax ``link://kind/name``. In
+templates you can also use ``_link(kind, name)``. You can add
+fragments (``#anchor``) and  query strings (``?key=value``) for extra
```

Comment:
This file is auto-generated, I’ll update the generation script myself (and do a slight rephrasing, and add a warning about editing this file).